elixir/apps/domain/lib/domain/tokens.ex

```diff
@@ -54,6 +54,14 @@ defmodule Domain.Tokens do
     end
   end
 
+  def list_tokens_by_type(type, %Auth.Subject{} = subject, opts \\ []) do
```

Comment:
Let's just add a hidden filter to `list_tokens/2` instead of this function.

Reply:
Isn't `list_tokens/2` private?  Are you saying you'd like me to make the `list_tokens/2` function public?

Reply:
@bmanifold no, just go to `Tokens.Token.Query.filters/0` and add a separate filter by `name` without a `title` (so it doesn't show up anywhere in the UI) . Then you can just do `list_tokens(subject, filter: [type: ...])` instead of calling this new function

Reply:
After working on this, I realized that `list_tokens_by_type` wasn't even being used.  I did add a new filter for type though, because I figured it would be useful.  I've removed the `list_tokens_by_type` function though.